plugins/hg4idea/testSrc/hg4idea/test/repo/HgRepositoryReaderTest.java

```diff
@@ -110,33 +216,35 @@ public void testLocalTags() {
     VcsTestUtil.assertEqualCollections(localTags, myLocalTags);
   }
 
-  @NotNull
-  private Collection<String> readBranches() throws IOException {
-    Collection<String> branches = new HashSet<>();
-    File branchHeads = new File(new File(myHgDir, "cache"), "branchheads-served");
-    String[] branchesWithHashes = FileUtil.loadFile(branchHeads).split("\n");
-    for (int i = 1; i < branchesWithHashes.length; ++i) {
-      String[] refAndName = branchesWithHashes[i].trim().split(" ");
-      assertEquals(2, refAndName.length);
-      branches.add(refAndName[1]);
-    }
-    return branches;
-  }
+  //@NotNull
+  //private Collection<String> readBranches() throws IOException {
+  //  Collection<String> branches = new HashSet<>();
+  //  File branchHeads = new File(new File(myHgDir, "cache"), "branchheads-served");
+  //  String[] branchesWithHashes = FileUtil.loadFile(branchHeads).split("\n");
+  //  for (int i = 1; i < branchesWithHashes.length; ++i) {
+  //    String[] refAndName = branchesWithHashes[i].trim().split(" ");
+  //    assertEquals(2, refAndName.length);
+  //    branches.add(refAndName[1]);
+  //  }
+  //  return branches;
+  //}
 
 
   public void testCurrentBookmark() {
-    assertEquals("B_BookMark", myRepositoryReader.readCurrentBookmark());
+    String bookmarkName = "bookmark_2";
+    updateRepoTo(bookmarkName);
+    assertEquals(bookmarkName, myRepositoryReader.readCurrentBookmark());
   }
 
-  @NotNull
-  private static Collection<String> readRefs(@NotNull File refFile) throws IOException {
-    Collection<String> refs = new HashSet<>();
-    String[] refsWithHashes = FileUtil.loadFile(refFile).split("\n");
-    for (String str : refsWithHashes) {
-      String[] refAndName = str.trim().split(" ");
-      assertEquals(2, refAndName.length);
-      refs.add(refAndName[1]);
-    }
-    return refs;
-  }
+  //@NotNull
```

Comment:
If the method is not needed anymore, please remove it. There is no need to comment it, since there is version control history.

Reply:
removed